### PR TITLE
Merge `1-10-stable` to `master`

### PR DESCRIPTION
**Pull request by igor-makarov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   [Elton Gao](https://github.com/gyfelton)
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#8649](https://github.com/CocoaPods/CocoaPods/issues/8649)
+* Generate the correct LD `-l` flags for XCFrameworks containing libraries
```
Comment (dnkoutso) [inline CHANGELOG.md:33]: nit this is off by one line

**Comment by dnkoutso** [comment]
I _think_ we need to upgrade integration specs repo too? I guess not cause the build is green?

**Review by dnkoutso — APPROVED** [review]

**Comment by igor-makarov** [comment]
There is an integration PR, I forgot to paste it here... Sorry!
https://github.com/CocoaPods/cocoapods-integration-specs/pull/300

**Comment by dnkoutso** [comment]
ah nice!

**Comment by igor-makarov** [comment]
@dnkoutso Fixed the changelog, updated the submodule! 💪🏻

**Pull request merged**